nubis/puppet/files/nat/confd/templates/iptables.tmpl

```diff
@@ -15,617 +6,198 @@ resources { 'firewall':
-    action  => 'accept',
-}
-
-firewall { '008 allow outbound tcp consul eth1':
-    chain       => 'OUTPUT',
-    outiface    => 'eth1',
-    destination => $vpc_cidr,
-    proto       => 'tcp',
-    dport       => [ '8300', '8301', '8302', '8400', '8500', '8600' ],
-    sport       => '1025-65535',
-    action      => 'accept',
-}
-
-firewall { '008 allow inbound tcp consul eth1':
-    chain     => 'INPUT',
-    iniface   => 'eth1',
-    source    => $vpc_cidr,
-    proto     => 'tcp',
-    tcp_flags => '! SYN,RST,ACK,FIN SYN',
-    sport     => [ '8300', '8301', '8302', '8400', '8500', '8600' ],
-    dport     => '1025-65535',
-    action    => 'accept',
-}
-
-# allow squid outbound
-firewall { '009 allow outbound udp squid':
-    chain    => 'OUTPUT',
-    outiface => 'eth1',
-    proto    => 'udp',
-    sport    => '3128',
-    dport    => '1025-65535',
-    action   => 'accept',
-}
-
-firewall { '009 allow inbound udp squid':
-    chain   => 'INPUT',
-    iniface => 'eth1',
-    source  => $vpc_cidr,
-    proto   => 'udp',
-    dport   => '3128',
-    sport   => '1025-65535',
-    action  => 'accept',
-}
-
-firewall { '009 allow outbound tcp squid':
-    chain    => 'OUTPUT',
-    outiface => 'eth1',
-    proto    => 'tcp',
-    sport    => '3128',
-    dport    => '1025-65535',
-    action   => 'accept',
-}
-
-firewall { '009 allow inbound tcp squid':
-    chain   => 'INPUT',
-    iniface => 'eth1',
-    source  => $vpc_cidr,
-    proto   => 'tcp',
-    dport   => '3128',
-    sport   => '1025-65535',
-    action  => 'accept',
-}
-
-# allow ntp outbound
-firewall { '010 allow outbound ntp':
-    chain       => 'OUTPUT',
-    outiface    => 'eth0',
-    proto       => 'udp',
-    sport       => '123',
-    dport       => '123',
-    destination => '0.0.0.0/0',
-    action      => 'accept',
-}
-
-firewall { '010 allow inbound ntp':
-    chain       => 'INPUT',
-    iniface     => 'eth0',
-    proto       => 'udp',
-    source      => '0.0.0.0/0',
-    sport       => '123',
-    destination => $vpc_cidr,
-    dport       => '123',
-    action      => 'accept',
-}
-
-firewall { "010 allow outbound ntp from unpriviledged port to ${vpc_cidr}":
-    chain       => 'OUTPUT',
-    outiface    => 'eth0',
-    proto       => 'udp',
-    source      => $vpc_cidr,
-    sport       => '1025-65535',
-    destination => '0.0.0.0/0',
-    dport       => '123',
-    action      => 'accept',
-}
-
-firewall { "010 allow inbound ntp from ${vpc_cidr} to unpriviledged port":
-    chain       => 'INPUT',
-    iniface     => 'eth0',
-    proto       => 'udp',
-    source      => '0.0.0.0/0',
-    sport       => '123',
-    destination => $vpc_cidr,
-    dport       => '1025-65535',
-    action      => 'accept',
-}
-
-# ssh rules
-firewall { '011 allow outbound ssh':
-    chain    => 'OUTPUT',
-    outiface => 'eth1',
-    proto    => tcp,
-    sport    => '22',
-    dport    => '1025-65535',
-    action   => 'accept',
+firewall { '998 INPUT logging':
```

Comment:
Yeah, I guess, I was trying to keep all INPUT rules together, and all FORWARD rules, but yeah, does make more sense.